src/test/java/TiTi/task/DeadlineTest.java

```diff
@@ -0,0 +1,22 @@
+package TiTi.task;
+
+import TiTi.task.Deadline;
+import org.junit.jupiter.api.Test;
```

Comment:
Regardless of the import order, I think there should be a new line separate these two imports as they are from different sources.